.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,43 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    # Workflow files stored in the
+    # default location of `.github/workflows`
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  # Maintain dependencies for go
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  # Maintain dependencies for dockerfile in the branches
+  - package-ecosystem: docker
+    directory: "/build/thirdparty/darwin"
+    target-branch: "tools-releases"
+    schedule:
+      interval: daily
+  - package-ecosystem: docker
+    directory: "/build/thirdparty/linux"
+    target-branch: "tools-releases"
+    schedule:
+      interval: daily
+
+  # Maintain dependencies for dockerfile scaffold in the projects
+  - package-ecosystem: docker
+    directory: "/project-v3"
```

Comment:
I think this should be `/testdata/project-v3`?

"You define the directory relative to the root of the repository for all ecosystems except GitHub Actions."
 
Reference: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directory

Reply:
+1